test/Project.toml

```
@@ -41,6 +41,7 @@ DynamicHMC = "2.1.6, 3.0"
 DynamicPPL = "0.15"
 FiniteDifferences = "0.10.8, 0.11, 0.12"
 ForwardDiff = "0.10.12"
+GalacticOptim = "1, 2"
```

Seems tests in the original PR used 0.4 with Julia 1.3:
```suggestion
GalacticOptim = "0.4, 1, 2"
```